lib/src/model/display_mode.dart

```diff
@@ -27,19 +29,25 @@ class DisplayMode {
     refreshRate: 0,
   );
 
+  Size get size => Size(width.toDouble(), height.toDouble());
+
   @override
   String toString() {
-    return '#$id ${width}x$height @ ${refreshRate.toInt()}Hz';
+    return '#$id $width×$height @ ${refreshRate.toInt()}Hz';
   }
 
   @override
-  bool operator ==(Object other) =>
-      identical(this, other) ||
-      other is DisplayMode &&
-          runtimeType == other.runtimeType &&
-          width == other.width &&
-          height == other.height &&
-          refreshRate == other.refreshRate;
+  bool operator ==(Object other) {
```

Comment:
Does this actually change the logic? Or is this due to a lint check? 🤔

Reply:
Just some styles updates actually. 😸